docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -13,32 +13,63 @@ Teleport FIPS mode works and how it can help your company to become FedRAMP auth
 Teleport includes FedRAMP and FIPS 140-2 features to support companies that sell into
 government agencies.
 
-| Control | Teleport Features |
+### Access Controls
+											
+|  Control | Teleport Features |
 | - | - |
-| [AC-02 Account Management]((=fedramp.control_url=)AC-2) | Audit events are emitted in the Auth Service when a user is created, updated, deleted, locked, or unlocked. |
-| [AC-03 Access Enforcement]((=fedramp.control_url=)AC-3) | Teleport Enterprise supports robust [Role-based Access Controls (RBAC)](../../access-controls/introduction.mdx) to: <br/>• Control which SSH nodes a user can or cannot access. <br/>• Control cluster level configuration (session recording, configuration, etc.) <br/>• Control which UNIX logins a user is allowed to use when logging into a server. |
+| [AC-02 Account Management]((=fedramp.control_url=)AC-02) | Audit events are emitted in the Auth Service when a user is created, updated, deleted, locked, or unlocked. |
+| [AC-03 Access Enforcement]((=fedramp.control_url=)AC-03) | Teleport Enterprise supports robust [Role-based Access Controls (RBAC)](../../access-controls/introduction.mdx) to: <br/>• Control which SSH nodes a user can or cannot access. <br/>• Control cluster level configuration (session recording, configuration, etc.) <br/>• Control which UNIX logins a user is allowed to use when logging into a server. |
+| [AC-07 Unsuccessful Logon Attempts]((=fedramp.control_url=)AC-07) |  Teleport Enterprise supports robust [Role-based Access Controls (RBAC)](../../access-controls/introduction.mdx) to: <br/>• Control which SSH nodes a user can or cannot access. <br/>• Control cluster level configuration (session recording, configuration, etc.) <br/>• Control which UNIX logins a user is allowed to use when logging into a server. |	Teleport supports two types of users: local and SSO-based accounts (GitHub, Google Apps, Okta, etc). For local accounts, by default, Teleport locks accounts for 30 minutes after 5 failed login attempts. For SSO-based accounts, the number of invalid login attempts and lockout time period is controlled by the SSO provider. |
```

Comment:
Looks like "Unix" is slightly more popular than "UNIX" in our docs, but I don't feel strongly either way.

```
teleport/docs on probakowski/keyboard-layout [$]
➜ ag --count Unix
pages/machine-id/reference/configuration.mdx:3
pages/includes/config-reference/ssh-service.yaml:1
pages/includes/metrics.mdx:2
pages/management/admin/labels.mdx:1
pages/management/admin/trustedclusters.mdx:1
pages/server-access/guides/host-user-creation.mdx:2
pages/server-access/guides/ssh-pam.mdx:7
pages/architecture/authorization.mdx:1
pages/reference/cli/tctl.mdx:6

teleport/docs on probakowski/keyboard-layout [$]
➜ ag --count UNIX
pages/access-controls/compliance-frameworks/fedramp.mdx:1
pages/access-controls/compliance-frameworks/soc2.mdx:1
pages/access-controls/guides/locking.mdx:1
pages/access-controls/sso.mdx:1
pages/includes/role-spec.mdx:1
pages/server-access/rbac.mdx:1
pages/architecture/authorization.mdx:1
pages/installation.mdx:1
```

Reply:
Updated for consistency